improver_tests/utilities/test_mathematical_operations.py

```diff
@@ -37,31 +37,28 @@
 import numpy as np
 from iris.tests import IrisTest
 
+from improver.utilities.cube_manipulation import sort_coord_in_cube
 from improver.utilities.mathematical_operations import Integration
 
-from ..ensemble_calibration.ensemble_calibration.helper_functions import (
-    set_up_temperature_cube)
+from ..set_up_test_cubes import set_up_variable_cube, add_coordinate
 
 
-def set_up_height_cube(height_points, cube=set_up_temperature_cube()):
-    """Create cube with added height dimension. By default the existing
-    set_up_temperature_cube utility is used."""
-    ascending = False
-    if np.all(np.diff(height_points) > 0):
-        ascending = True
+def set_up_height_cube(height_points, ascending=True):
```

Comment:
Can you please make this a private function. It is specifically designed for these tests, i.e. accepts only 3 height-points without checks/specifying this, so should probably only be used here.

Reply:
OK